templates/CRM/UF/Form/FieldRadioOptionsPerLine.tpl

```diff
@@ -0,0 +1,20 @@
+<div class="crm-section editrow_{$profileFieldName}-section form-item" id="editrow-{$rowIdentifier}">
+  <div class="label option-label">{$formElement.label}</div>
+  <div class="content">
+    <div class="crm-multiple-checkbox-radio-options crm-options-per-line" style="--crm-opts-per-line:{$field.options_per_line};">
+      {foreach name=outer key=key item=item from=$formElement}
+        {if is_array($item) && array_key_exists('html', $item)}
+          <div class="crm-option-label-pair" >{$formElement.$key.html}</div>
+        {/if}
+      {/foreach}
+    </div>
+    {* Include the edit options list for admins *}
+    {if $formElement.html|strstr:"crm-option-edit-link"}
+      {$formElement.html|regex_replace:"@^.*(<a href=.*? class=.crm-option-edit-link.*?</a>)$@":"$1"}
+    {else}
+      {$formElement.html}
```

Comment:
This seems like it might duplicate things? I don't see this in the fields.tpl nor in dynamic.tpl previously

Reply:
or is this just the wrench?

Reply:
@seamuslee001 yeah so what was happening is that because the radio options were not being laid out as desired the tpl was iterating through & rendering the options html not the radio group html - and then using that to re-add some of the radio group html (but was dropping the clear when the edit was not in play)

Although this PR does mostly get us there I wound up concluding there is a better approach - ie update the renderer to add the classes - see https://github.com/civicrm/civicrm-core/pull/31706 (as of writing the PR name is not a good description of what it does)